yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -670,6 +670,21 @@ def pixelize_aitoff(np.float64_t[:] azimuth,
         ymax = fmax(ymax, y)
         # Now we have the (projected rectangular) bounds.
 
+        # special cases where the projection of the cell isn't
+        # bounded by the rectangle (in image space) that bounds its corners.
+        # We take a generous margin of error by making the rectangle 3 times as
+        # large (or tall), which is enough to accomodate even extremely coarse grids
+        if xmin * xmax < 0.0:
```

Comment:
I think I agree with this change, but a question: would we have this issue if we did everything in coords from 0..180 and 0..360?

Reply:
Yes. The algorithm is the problem, not the coordinate system. In fact the coordinates we use here are actually more convenient to workaround these special cases.